bundles/org.openhab.core/src/main/java/org/eclipse/smarthome/core/library/unit/SmartHomeUnits.java

```diff
@@ -82,8 +84,8 @@
 
     // Alphabetical ordered by Unit.
     public static final Unit<Acceleration> METRE_PER_SQUARE_SECOND = addUnit(Units.METRE_PER_SQUARE_SECOND);
-    public static final Unit<Acceleration> STANDARD_GRAVITY = new TransformedUnit<>("gₙ",
```

Comment:
Don't we loose the symbol "g" when applying this.

Reply:
I think so. Rest looks good IMHO.